cypress/entrypoint.sh

```diff
@@ -19,8 +19,10 @@ if [ $? -eq 0 ]; then
         php /var/www/html/occ user:add --password-from-env user2
         php /var/www/html/occ app:enable viewer
         php /var/www/html/occ app:enable text
+        php /var/www/html/occ app:enable assistant
```

Comment:
I think this was for some way of running isolated cypress tests locally, but never used that script myself. @max-nextcloud Do you happen to know if we still make use of that?

Reply:
I don't remember it. Looking at the log of that file indicates that it was introduced by @susnux in f62c29be.

Reply:
Was just for local tests, but nowadays we could use `@nextcloud/cypress/docker` which also works locally :)